tools/generator/src/Generator+AddTargets.swift

```diff
@@ -471,7 +466,30 @@ private extension Path {
     }
 }
 
-extension Outputs {
+private extension ConsolidatedTarget {
+    var generatesSwiftHeader: Bool {
+        guard isSwift else {
+            return false
+        }
+
+        return targets.values.allSatisfy { target in
+            return target.buildSettings["SWIFT_OBJC_INTERFACE_HEADER_NAME"]
+                != .string("")
```

Comment:
Can `target.buildSettings["SWIFT_OBJC_INTERFACE_HEADER_NAME"]` ever be nil? If so, this check should account for that.

Reply:
Yes, good catch.